src/pages/home/report/ReportActionsList.js

```diff
@@ -150,7 +156,8 @@ function ReportActionsList({
         }
         return cacheUnreadMarkers.get(report.reportID);
     };
-    const reportScrollManager = useReportScrollManager();
+    const platform = getPlatform();
```

Comment:
Definitely we should not use this API, and instead should extract `getInitialNumToRender` to a separate module that has web and native implementations.

Reply:
👍🏻